skrifa/src/scale/scaler.rs

```diff
@@ -122,22 +123,30 @@ impl<'a> ScalerBuilder<'a> {
     pub fn build(mut self, font: &impl TableProvider<'a>) -> Scaler<'a> {
```

Comment:
I wonder if the scaler builder should let me set what outline tables I prefer in prioritized order, probably defaulting to [glyf, cff2, cff]?

Reply:
I'm going to agree with earlier Rod in saying that skrifa users probably shouldn't care about this level of detail. We should also likely be using `sfntVersion` to determine priority instead (that is, prefer CFF2->CFF->glyf when the version is OTTO and glyf->CFF2->CFF otherwise). We don't yet have enough information to do that here. See #485